addons/openvpn/openvpn-client-dep.yaml

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+{{ if not .Cluster.Network.KonnectivityEnabled }}
```

Comment:
could not we just skip the whole openvpn addon in the addont controller if Konnectivity is enabled?

Reply:
You are right, it probably would be nicer to skip it in the controller, I can change that.